src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -197,6 +195,96 @@ def end_value(device)
         return "" if device.region.empty?
         device.region.end
       end
+
+      DEVICE_ICONS = {
+        disk:      Icons::HD,
+        dasd:      Icons::HD,
+        multipath: Icons::MULTIPATH,
+        partition: Icons::HD_PART,
+        raid:      Icons::RAID,
+        lvm_vg:    Icons::LVM,
+        lvm_lv:    Icons::LVM_LV
+      }
+
+      # Table icon for the device
+      #
+      # @see DEVICE_ICONS
+      #
+      # @param device [Y2Storage::BlkDevice]
+      # @return [Yast::Term] icon
+      def type_icon(device)
+        return type_icon(device.plain_blk_device) if device.is?(:lvm_pv)
+
+        type = DEVICE_ICONS.keys.find { |k| device.is?(k) }
+        icon = type.nil? ? Icons::DEFAULT_DEVICE : DEVICE_ICONS[type]
+
+        small_icon(icon)
+      end
+
+      DEVICE_LABELS = {
+        disk:          _("DISK"),
+        dasd:          _("DISK"),
```

Comment:
Why the usage of UPCASE in terms like "DISK" or "PARTITION"?

On the other hand, maybe a comment to translators to advise to keep this sort (to be displayed in a column of a big list).

Reply:
Well MD RAID and LVM are correctly capitalized in all upcase because they are abbreviations of names (e.g. Multiple Device Redundant Array of Independent Disks). So not the same case. 

On the other hand, I don't see how `PARTITION` can end up being used. And maybe the same applies to others. So please add a comment to those values that are there just for completeness but that are impossible to reach.